jvb/src/main/java/org/jitsi/videobridge/octo/OctoEndpointMessageTransport.java

```diff
@@ -78,6 +78,7 @@ public BridgeChannelMessage addReceiver(@NotNull AddReceiverMessage message)
         // only those that reference an endpoint local to this bridge.
         if (endpoint instanceof Endpoint)
         {
+            // TODO Implement source name based receiver constraints for Octo
```

Comment:
This is actually not necessary. Remote endpoints can only subscribe to local endpoint. I know this is very confusing, I'll try and sketch a diagram

Reply:
The way I understand this code is that it registers an Octo JVB as a receiver of a local endpoint with specific constraints. I don't understand why that would not be necessary in the source name world - a jvb will register for a given source, no?

Reply:
well it wouldn't be exactly here, because needs a new message type?

Reply:
I've skipped OctoV1 for multi-stream, because this gives one more combination to test and maintain. Let me know if that's ok.